RULES.md

```diff
@@ -429,3 +430,14 @@ At least one of the fields `is_producer`, `is_operator`, or `is_authority` shoul
 
 #### References:
 * [attributions.txt specification](https://gtfs.org/reference/static#attributionstxt)
+
+<a name="RouteNameMissingOrEmptyNotice"/>
+
+### RouteNameMissingOrEmptyNotice
+
+One of the following fields is missing or empty: 
+- `routes.route_short_name` 
+- `routes.route_long_name`
+
+#### References:
+* [Python validator original implementation](https://github.com/google/transitfeed/blob/master/transitfeed/route.py)
```

Comment:
I am unsure if we need this check. Here is some statistics from all prod feeds at Google. 

- 45% give both names
- 35% give ONLY long name
- 20% give ONLY short name

Less than a half routes give both names and I think that makes sense. Real-world transit routes may indeed have just one name. If we issue warnings, that may push some GTFS providers to invent short or long names that never existed in the real world just to satisfy the validator.

Ceterum censeo Python validator should be taken with a grain of salt. It was never used on a wide range of feeds and before porting a validation rules from Python, we should have evidence that it's a meaningful rule